tigerpath/static/app/js/mobile_banner.js

```diff
@@ -0,0 +1,7 @@
+window.addEventListener("resize", function() {
+  if (window.innerWidth <= 991) {
```

Comment:
Should `991` really be hardcoded or is there some existing parameter that is used by Bootstrap that can be read?
Either way, it should probably be factored out.

Reply:
You're right, the JavaScript code is superfluous. Everything is now done in css

Reply:
The fractional pixel is standard for bootstrap: https://getbootstrap.com/docs/4.1/layout/overview/

Unfortunately CSS doesn't have variables, so one thing we can look into adopting in the future is SASS